lib/Config.js

```diff
@@ -183,13 +202,22 @@ class Config {
                     this.utapi.redis.host = config.utapi.redis.host;
                     this.utapi.redis.port = config.utapi.redis.port;
                 }
+            } else {
+                // `redis` is a required property of UtapiClient
+                config.utapi.redis = this.localCache;
```

Comment:
You should throw an assertion here, if `config.utapi` is set, it should have a redis section. It's a bad idea to use localcache for fallback.